### PR TITLE
Android menu working

### PR DESCRIPTION
**Pull request by ThatFlint**
The android app now has a functional UI

### CODE REVIEW

**Comment by NilsHullegien** [comment]
Code looks good, App deploys. I approve of this code.

File: ContextApp/src/main/java/com/github/migi_1/ContextApp/HelloActivity.java [hunk 1]
```diff
@@ -84,5 +105,21 @@ public void onCreate(Bundle savedInstanceState) {
         public Main getMain() {
             return application;
         }
+        
+        /**
+         * Makes sure buttonpresses are logged.
+         * @param butt Button
+         * @param str String
```
Comment (LMiljak) [inline ContextApp/src/main/java/com/github/migi_1/ContextApp/HelloActivity.java]: Not really a valid parameter description.

**Comment by LMiljak** [comment]
Just one very minor remark.
Other than that, it's ready to be merged.

**Comment by LMiljak** [comment]
:+1:  Merging

**Pull request merged**